### ISSUE TITLE
Will Rosetta use BluePad32 for wireless joysticks?

### ISSUE THREAD

**Issue by dglaude** [drflamemontgomery/Rosetta#3]
https://gitlab.com/ricardoquesada/bluepad32

**Comment by drflamemontgomery** [comment]
Yes, Rosetta will be using BluePad32 for wireless gamepads at least for prototyping. In future, I might create my own Bluetooth library for better support with the Rosetta remapping functionality. But for now, I am using BluePad32 because it works out of the box.
